app/queries/servers/group.ts

```diff
@@ -63,3 +63,7 @@ export const deleteGroupMembershipById = (database: Database, id: string) => {
 export const deleteGroupTeamById = (database: Database, id: string) => {
     return database.collections.get<GroupTeamModel>(GROUP_TEAM).find(id).then((model) => model.destroyPermanently());
 };
+
+export const deleteGroupChannelById = (database: Database, id: string) => {
+    return database.collections.get<GroupChannelModel>(GROUP_CHANNEL).find(id).then((model) => model.destroyPermanently());
```

Comment:
When you say **handles the websocket events for groups being associated / dissociated with Channels**, it does not mean that the Group itself is being removed and that's why you are not deleting the related records in the other tables? Correct ?

Reply:
Yup - this is only for the `GroupChannel` relationship.